### PR TITLE
docs(aio): add hero “Zero” to toh-6

### PR DESCRIPTION
**Pull request by wardbell**
Update to in-memory-web-api should handle id=0.
Make sure this works by having a hero with id=0 in ToH.
Coincidentally delete lingering dead app/ and dart folders
Todo: fix ToH images (which have to do anyway)

<hr>

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

**Review by petebacondarwin — APPROVED** [review]
I can approve the package.json change.
I am not sure if I am qualified to say if the deleted files should indeed be deleted...

**Review by Foxandxss — APPROVED** [review]

**Comment by wardbell** [comment]
The Travis failure sure looks like a flake. This PR is purely a documentation change (the `.ts` and `.html` changes are to examples as is the `yarn.lock` change which consists of a version update to the example-oriented in-memory-web-api.

I'll restart build and hope for the best.

File: aio/content/examples/toh-pt1/app/app.component.1.ts [hunk 1]
```diff
@@ -1,44 +0,0 @@
-import { Component } from '@angular/core';
```
Comment (petebacondarwin) [inline aio/content/examples/toh-pt1/app/app.component.1.ts]: @wardbell this file is actually referenced in the `tutorial/toh-pt1.md` doc.

File: aio/content/examples/toh-pt3/app/app.component.1.html [hunk 2]
```diff
@@ -1,12 +0,0 @@
-<h1>{{title}}</h1>
```
Comment (petebacondarwin) [inline aio/content/examples/toh-pt3/app/app.component.1.html]: @wardbell this document is actually referenced in the `tutorial/toh-pt3.md` doc.

File: aio/content/examples/toh-pt3/app/hero-detail.component.1.ts [hunk 3]
```diff
@@ -1,35 +0,0 @@
-// #docplaster
```
Comment (petebacondarwin) [inline aio/content/examples/toh-pt3/app/hero-detail.component.1.ts]: @wardbell this document is actually referenced in the tutorial/toh-pt3.md doc.

**Pull request merged**